express-notes-api/index.js

```diff
@@ -0,0 +1,140 @@
+const express = require('express');
+const fs = require('fs');
+const app = express();
+
+app.get('/api/notes', (req, res) => {
+  const noteArr = [];
+  fs.readFile('data.json', 'utf-8', (err, data) => {
+    if (err) {
+      console.error(err);
+      res.status(500).json({ error: 'Unexpected error!' });
+    }
```

Comment:
Be sure to `return` or include `else` to prevent the server from attempting to respond more than once to the same request.